chromehangs/index.html

```diff
@@ -9,9 +9,9 @@
     <script type="text/javascript" src="https://code.jquery.com/jquery.js"></script>
     <script type="text/javascript" src="https://code.jquery.com/ui/1.10.4/jquery-ui.js"></script>
     <script type="text/javascript" src="chromehangs.js"></script>
-    <script src="http://jquery-csv.googlecode.com/git/src/jquery.csv.js"></script>
+    <script src="//jquery-csv.googlecode.com/git/src/jquery.csv.js"></script>
```

Comment:
Yes, it just matches the top-level URI scheme.  It could just use `https` all the time if you prefer. Using `//` keeps the current behaviour if you access telemetry.m.o via http, while fixing the https case.